text/0079-cdk-2.0.md

```diff
@@ -200,6 +178,32 @@ auditable by users, who will naturally retain the ability to opt out of this fea
 
 [version reporting]: https://docs.aws.amazon.com/cdk/latest/guide/cli.html#version_reporting
 
+## Releasing to Package Managers
+
+`aws-cdk-lib` will be released to the various package managers under the following package name.
+
+| Github & NPM | Maven Central | PyPI | NuGet |
+| -- | -- | -- | -- |
+| `aws-cdk-lib` | `software.amazon.awscdk.lib` | `aws-cdk.lib` | `Amazon.CDK.Lib` |
```

Comment:
I think python should be `aws-cdk-lib` (similar to npm if possible).

Reply:
Makes sense